### PR TITLE
[PATCH v3] changelog: updates for odp v1.19.0.0

### PR DESCRIPTION
**Pull request by Bill-Fischofer-Linaro**
Add updates for v1.19.0.0 (Tiger Moth Final Release)

Signed-off-by: Bill Fischofer <bill.fischofer@linaro.org>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
+=== Summary of Changes
+ODP v1.19.0.0 is the official Tiger Moth final release. It incorporates final
+implementation changes and bug fixes and completes the Tiger Moth ODP
+development cycle.
+
+==== APIs
+No changes for this release. The Tiger Moth API was frozen in ODP v1.18.0.0.
+
+==== C++ Test Improvements
+The $$C++$$ test included in the ODP validation suite now uses `cout` instead
```
Comment (bogdanPricope) [inline CHANGELOG]: Do we need the '$$' in '$$C++$$' or... ?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,72 @@
+== OpenDataPlane (1.19.0.0)
```
Comment (muvarov) [inline CHANGELOG]: if we do not change API here it has to be 1.18.0.2

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,72 @@
...
+The $$C++$$ test included in the ODP validation suite now uses `cout` instead
+of `printf()` to ensure that $$C++$$ is being used to compile it.
+
+==== Queue and Scheduler Configuration
+For the ODP Reference Implementation, The `config/odp-linux-generic.conf` file
+is extended with sections to control the default and maximum sizes for basic
+queues, and the priority spread used by the scheduler for scheduled queues.
+
+==== Runtime Default `config` File Read Order Improvements
+For the ODP Reference Implementation, if an application-provided `config` file
```
Comment (bogdanPricope) [inline CHANGELOG]: "For the ODP Reference Implementation, the default values of application-provided `config` file (if used) overrides the values provided by the built-in `config/odp-linux-config.conf` file."

File: CHANGELOG (hunk 1 above)
Comment (lumag) [reply CHANGELOG]: @bogdanPricope @Bill-Fischofer-Linaro https://github.com/asciidoctor/asciidoctor/issues/1208

Comment (Bill-Fischofer-Linaro) [reply CHANGELOG]: Thanks for that pointer. I'll use it, though {cpp} in the text still looks a bit strange"

File: CHANGELOG (hunk 2 above)
Comment (Bill-Fischofer-Linaro) [reply CHANGELOG]: OK.

**Comment by bogdanPricope** [comment]
Reviewed-by: Bogdan Pricope <bogdan.pricope@linaro.org>

**Comment by joseppc** [comment]
Maybe we should also mention that when building the DPDK PKTIO, we now need to point to the installation path rather than the build directory.

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v3:
* Rebased to latest master
* Updated to reflect merge of Bug 3690 fix

**Pull request closed without merging**